src/components/Tabs/TabContainer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+const Div = styled.div`
+  ${({ vertical }) => {
+    if (vertical) {
+      return css`  
+    width: 100%;
+    display: table;
+  `;
+    }
+  }}`;
+
+export default class TabContainer extends React.Component {
```

Comment:
`TabContainer` component name collides with `TabContainer` internal component under `Tab`. Need to be careful with generic names like `Container`. This appears to be an element that contains tabs; perhaps `TabList` is a better name?

Reply:
for sure that could help to get rid of collisions